elephant/utils.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+"""
+Elephant-wide utility functions.
+
+:copyright: Copyright 2015-2019 by the Elephant team, see AUTHORS.txt.
+:license: Modified BSD, see LICENSE.txt for details.
+"""
+
+
+import quantities as pq
+
+
+def check_quantities(var, var_name):
+    """
+    Checks input `var` data for having the type of `pq.Quantity`.
+
+    Parameters
+    ----------
+    var
+        input argument
+    var_name : str
+        variable name
+
+    Raises
+    -------
+    AssertionError
+        If the input `var` is not a `pq.Quantity`
+    """
+    assert isinstance(var, pq.Quantity), \
```

Comment:
I don't see a merit in defining this function.
Does this increase readability/maintainability of code so much?

Reply:
Yes for maintainability and consistency. It does not improve readability.
Reason: instead of typing the same thing each time in each function.

Reply:
Forgot to mention. I'll add this function check to all elephant's public functions.

Reply:
Then you should define this function in that PR, not here.